src/main/java/org/broadinstitute/hellbender/tools/walkers/concordance/Concordance.java

```diff
@@ -0,0 +1,244 @@
+        // otherwise we move the eval forward without giving it a diagnosis
+        while (variantContextComparator.compare(variant, currentTruthVariant.get()) > 0) {
+            if (currentTruthVariant.get().isSNP()) { snpFalseNegatives++; } else { indelFalseNegatives++; }
+            if (truthIterator.hasNext()) {
+                currentTruthVariant = getNextNonSVTruthVariant();
+                if (! currentTruthVariant.isPresent()) {
+                    return;
+                }
+            } else {
+                exhaustedTruthVariants = true;
+                return;
+            }
+        }
+
+        // case 2: the position of the two variants match
+        if (variantContextComparator.compare(variant, currentTruthVariant.get()) == 0) {
+            // do the genotypes match too?
+            if (variant.isFiltered()) {
+                if (variant.isSNP()) { snpFalseNegatives++; } else { indelFalseNegatives++; }
+                writeThisRecord(new VariantStatusRecord(variant, FALSE_NEGATIVE, evalSampleName), variant);
+            } else if (allelesMatch(variant, currentTruthVariant.get())) {
+                if (variant.isSNP()) { snpTruePositives++; } else { indelTruePositives++; }
+                writeThisRecord(new VariantStatusRecord(variant, TRUE_POSITIVE, evalSampleName), variant);
+            } else {
+                // the eval variant matches truth's position but their alleles don't match
+                // e.g. genotype or alleles don't match
+                if (variant.isSNP()) {
+                    snpFalsePositives++;
+                    snpFalseNegatives++;
+                } else {
+                    indelFalsePositives++;
+                    indelFalseNegatives++;
+                }
+                writeThisRecord(new VariantStatusRecord(variant, FALSE_POSITIVE, evalSampleName), variant);
+            }
+
+            if (truthIterator.hasNext()) {
+                currentTruthVariant = getNextNonSVTruthVariant();
+            } else {
+                exhaustedTruthVariants = true;
+            }
+
+            return;
+        }
+
+        // case 3: truth got ahead of eval; the eval variant must be a false positive if not filtered
+        if (variantContextComparator.compare(variant, currentTruthVariant.get()) < 0) {
+            if (variant.isNotFiltered()) {
+                if (variant.isSNP()) { snpFalsePositives++; } else { indelFalsePositives++; }
+                writeThisRecord(new VariantStatusRecord(variant, FALSE_POSITIVE, evalSampleName), variant);
+            }
+
+            // we don't care about true negatives - don't record them in the table
+            return;
+        }
+    }
+
+    // ugly code alert
+    // this method just moves the iterator forward but also ensures that it skips SV's
+    // assumes that the caller already checked the truth iterator has next
+    // TODO: extend the truth iterator such that the iterator.next() automatically skips SVs
```

Comment:
You can implement this `TODO` by using a `FilterIterator` (from apache common 4, *not* 2): `final Iterator<VariantContext> svOnlyTruthIterator = new FilterIterator<>(truthIterator, vc -> vc.isSymbolicOrSV());`  This method then becomes superfluous.